cpp/daal/src/algorithms/k_nearest_neighbors/bf_knn_classification_predict_kernel.h

```diff
@@ -35,12 +35,25 @@ namespace internal
 {
 using namespace daal::data_management;
 
+enum class PairwiseDistanceType
+{
+    minkowski,
+    chebychev
+};
+
+struct KernelParameter : bf_knn_classification::Parameter
+{
+    bf_knn_classification::VoteWeights voteWeights = bf_knn_classification::VoteWeights::voteUniform;
+    PairwiseDistanceType pairwiseDistance          = PairwiseDistanceType::minkowski;
+    double minkowskiDegree                         = 2.0;
```

Comment:
Shouldn't it be `const`?

Reply:
It should be changeable as far as we want to set this parameter from new api.